internal/engine/eval/vulncheck/vulncheck.go

```diff
@@ -57,30 +57,42 @@ func NewVulncheckEvaluator(_ *pb.RuleType_Definition_Eval_Vulncheck, pbuild *pro
 }
 
 // Eval implements the Evaluator interface.
-//
-//nolint:gocyclo
 func (e *Evaluator) Eval(ctx context.Context, pol map[string]any, res *engif.Result) error {
```

Comment:
Do we need to break out getVulnerableDependencies and Eval?  I think moving 93-133 into a separate method definitely makes sense, thank you!

Reply:
I suppose we don't actually need to break them up. Thought it would be more readable since it allows for one to easily see what `Eval` is doing at a higher level (i.e getting vulnerable dependencies and returning an err if any are found). 

Happy to re-consolidate `getVulnerableDependencies` back into the `Eval` though.

Reply:
I'll buy that.